conda_build/skeletons/cran.py

```diff
@@ -143,6 +148,33 @@
 else
   mkdir -p $PREFIX/lib/R/library/{cran_packagename}
   mv * $PREFIX/lib/R/library/{cran_packagename}
+
+  if [[ $target_platform == osx-64 ]]; then
+    pushd $PREFIX
+      for libdir in lib/R/lib lib/R/modules lib/R/library lib/R/bin/exec sysroot/usr/lib; do
+        pushd $libdir || exit 1
+          for SHARED_LIB in $(find . -type f -iname "*.dylib" -or -iname "*.so" -or -iname "R"); do
+            echo "fixing SHARED_LIB $SHARED_LIB"
+            install_name_tool -change /Library/Frameworks/R.framework/Versions/3.5.0-MRO/Resources/lib/libR.dylib "$PREFIX"/lib/R/lib/libR.dylib $SHARED_LIB || true
+            install_name_tool -change /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libR.dylib "$PREFIX"/lib/R/lib/libR.dylib $SHARED_LIB || true
+            install_name_tool -change /usr/local/clang4/lib/libomp.dylib "$PREFIX"/lib/libomp.dylib $SHARED_LIB || true
+            install_name_tool -change /usr/local/gfortran/lib/libgfortran.3.dylib "$PREFIX"/lib/libgfortran.3.dylib $SHARED_LIB || true
```

Comment:
the libgfortran so version changes in recent GCC (it's on 5 in gcc 8.2).  We're still on gcc 4.8 for mac, so it doesn't matter here yet, but should we constrain libgfortran somehow in light of this?

Reply:
If I need to pin libgfortran I'll do it in r-base only I think.